### PR TITLE
Legion can convert Combat Armors too now

### PR DESCRIPTION
**Pull request by BrotherHangyul**
## About The Pull Request

Legion can now properly convert mark one and two combat armour to Legion colours using the crafting menu.

## Why It's Good For The Game

Variety.

Less spray shitters.

Also Ultrabus asked.

I tested it, should work.

## Pre-Merge Checklist

- [x] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.

## Changelog

:cl:
add: You can now convert mark one and two combat armours to legion.
/:cl:

### CODE REVIEW

File: code/modules/clothing/head/f13factionhead.dm [hunk 1]
```diff
@@ -497,6 +497,18 @@
 	icon_state = "legion-heavy"
 	item_state = "legion-heavy"
 	armor = list("melee" = 60, "bullet" = 45, "laser" = 60, "energy" = 15, "bomb" = 45, "bio" = 60, "rad" = 30, "fire" = 60, "acid" = 20, "wound" = 50)
+	
+/obj/item/clothing/head/helmet/f13/combat/legion
```
Comment (BadAtThisGame302) [inline code/modules/clothing/head/f13factionhead.dm:501]: These already exist, don't add something already in the code.

Comment (BrotherHangyul) [reply code/modules/clothing/head/f13factionhead.dm:501]: They don't exist, I literally checked through f13factionhead.

Comment (BadAtThisGame302) [reply code/modules/clothing/head/f13factionhead.dm:501]: image

Comment (BadAtThisGame302) [reply code/modules/clothing/head/f13factionhead.dm:501]: Bottom of f13factionhead.dm

Comment (BrotherHangyul) [reply code/modules/clothing/head/f13factionhead.dm:501]: removed em 👍

**Comment by OliOliOnsiPree** [comment]
-1.

**Comment by Legate-Maxson** [comment]
I will remove the ability for the legion to plate their armor if this is merged -1

**Comment by BrotherHangyul** [comment]
why??? it's a reskin...

**Comment by Legate-Maxson** [comment]
> why??? it's a reskin...

NCR cant do it. Its in the Faction DM so it can be plated. No other faction can do this. Give it to everyone or none.

**Comment by Legate-Maxson** [comment]
> 

It cannot

**Comment by BrotherHangyul** [comment]
I'm closing this just for like a half hour so I can make a new PR that gives the armour to everyone (enclave/ncr, bos already has this shit)

**Pull request closed without merging**